app/models/heroku_application.rb

```diff
@@ -0,0 +1,9 @@
+class HerokuApplication < ApplicationRecord
```

Comment:
nit: It'd probably make sense to just add some defensive validations to ensure that any application we create has a name, oauth token, grace period defined.

Also would be kinda good to be able to test that the key works (ie. can authenticate) before it tries to actually trigger a restart.

Not blockers to getting this configured though and running it.

Reply:
👍 added those to https://github.com/lockstep/heroku_memory_capitulator/issues/2